terraform/modules/alarm/main.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_cloudwatch_metric_alarm" "error_rate_alarm" {
+  alarm_name          = var.alarm_name
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "2"
+  threshold           = "10"
+  alarm_description   = var.alarm_description
+  treat_missing_data  = "notBreaching"
+  alarm_actions       = [var.alarm_action]
+  ok_actions          = [var.alarm_action]
```

Comment:
Are these two meant to be the same?

Reply:
Yep, these just tell you what system to notify, so we want it to notify the SNS topic, but it will notify it differently depending on it being alarm or ok

Reply:
This was on @oskarwilliams378's example from lner